plugins/openshift-image-registry/src/components/OcirImages/OcirImageSidebar.tsx

```diff
@@ -133,7 +133,7 @@ export const OcirImageSidebar = ({
             </Typography>
             {imageStream.tags?.length
               ? imageStream.tags.map((tag: string, i: number) => (
-                  <Chip size="small" label={tag} key={i} />
+                  <Chip size="small" label={tag} key={i} /> // NOSONAR
```

Comment:
similar, can we make use of a tag here over the index?

Reply:
Oh thanks GitHub, missed that before. Used the tag here now also as key.